docs/docs/userguide/configuration/BITBUCKET_CONFIGURATION.md

```diff
@@ -148,4 +149,39 @@ bitbucket_elasticsearch_cpu      = "0.25"
 bitbucket_elasticsearch_mem      = "1Gi"
 bitbucket_elasticsearch_storage  = 10
 bitbucket_elasticsearch_replicas = 2
+```
+
+## Dataset restore
+To restore the dataset into the newly created instance, uncomment the following lines and provide all necessary parameters.
+
+### Database Snapshot Identifier
+
+`bitbucket_db_snapshot_identifier` sets the identifier for the DB snapshot to restore from. If you do not specify a value, no AWS RDS snapshot is used.
+
+```terraform
+bitbucket_db_snapshot_identifier = "<SNAPSHOT_IDENTIFIER>"   # e.g. "my-snapshot"
+```
+
+!!! info "The AWS RDS snapshot must be in the same region and account as the RDS instance."
+
+    You also need to provide the master user credentials (`bitbucket_db_master_username` and `bitbucket_db_master_password`) that match the snapshot.
+
+!!! tip "Optimise the restore performance."
+
+    To obtain the best performance, configure Bitbucket RDS that match the snapshot including `bitbucket_db_instance_class` and `bitbucket_db_allocated_storage`.
+
+### Database Master Username
+
+'bitbucket_db_master_username' sets the username for the RDS master user. If you do not specify a value, username is "postgres".
```

Comment:
Should we make it more explicit that if the user uses snapshot, they should also make sure that the username and password should match the database snapshot?